deployments/apps/main.tf

```diff
@@ -0,0 +1,83 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    digitalocean = {
+      source  = "digitalocean/digitalocean"
+      version = ">= 2.0.0"
+    }
+  }
+}
+
+provider "digitalocean" {}
+
+resource "digitalocean_app" "server-api" {
+  spec {
+    name = "server-api"
+    region = "blr"
+
+    domain {
+      name = "api.asyncapi.com"
```

Comment:
Is this enough for configuring a domain + SSL management?

Reply:
If you add the DO servers then it can be managed there itself I guess. But as we are using subdomains, I guess we will have to point CNAME to the do server.

![image](https://github.com/asyncapi/server-api/assets/100484401/832cfbb2-065d-4e2c-8886-b2c38fba92eb)
![image](https://github.com/asyncapi/server-api/assets/100484401/185f5e31-3a73-4f0b-97c2-569ff9e09fa2)

Reply:
http://server-api.ashish-padhy.co/v1/help/validate
Like this

Reply:
https://www.digitalocean.com/community/questions/app-aplication-in-laravel

Reply:
> If you add the DO servers then it can be managed there itself I guess. But as we are using subdomains, I guess we will have to point CNAME to the do server.
> 
> ![image](https://github.com/asyncapi/server-api/assets/100484401/832cfbb2-065d-4e2c-8886-b2c38fba92eb)
> ![image](https://github.com/asyncapi/server-api/assets/100484401/185f5e31-3a73-4f0b-97c2-569ff9e09fa2)
> 

api.asyncapi.com DNS are pointing already to Digital Ocean ones.
The point is, you should need a eay to tell Digital Ocean "this" is the domain for this service, please manage the SSL, etc

Reply:
As you pointed out with https://www.digitalocean.com/community/questions/app-aplication-in-laravel, you should need to explicitly link a domain to your app in Digital Ocean.
We should replicate that into Terraform

Reply:
This is only applicable for domains, not sub-domain though 🤔

Reply:
Yeah, http://server-api.ashish-padhy.co/v1/help/validate this one.
However, it won't work now as my digital ocean credits have expired.

Reply:
Awesome! Thanks for testing!